pkg/cloudprovider/providers/photon/photon.go

```diff
@@ -66,22 +65,23 @@ type PCCloud struct {
 	// in the environment per current Photon Controller deployment methodology.
 	projID string
 	cloudprovider.Zone
+	photonClient *photon.Client
```

Comment:
I don't know exactly how things work. How many threads might be accessing the client? I'm slightly worried that the client isn't fully safe with respect to multiple threads. We can fix that if needed--I'm curious if it's urgent.